File: SimuladoAtividade1/src/Usuario.java
//Matheus Coelho Rocha Pinto - 20.00391-9

public class Usuario {
    private String nome;
    private String senha;
    private String email;

    public Usuario(String nome, String senha, String email) {
        this.nome = nome;
        this.senha = senha;
        this.email = email;
    }

    public void visualizarCliente() {
        System.out.println("Dados do Cliente:");
        System.out.println("Nome:" + nome);
        System.out.println("CPF:" + senha);
        System.out.println("E-mail:" + email);
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getSenha() {
        return senha;
    }

    public void setSenha(String senha) {
        this.senha = senha;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

}

File: T4/src/models/usuario.py
class Usuario:
    def __init__(self, id, nome_usuario, email, senha) -> None:
        self._id = id
        self._nome_usuario = nome_usuario
        self._email = email
        self._senha = senha
    
    def get_nome_usuario(self):
        return self._nome_usuario
    def set_nome_usuario(self, nome_usuario):
        self._nome_usuario = nome_usuario
    
    def get_email(self):
        return self._email
    def set_email(self, email):
        self._email = email
    
    def get_senha(self):
        return self._senha
    def set_senha(self, senha):
        self._senha = senha
    
    def __str__(self) -> str:
        return f'Usuario: Nome:{self._nome_usuario} - Email:{self._email}'
    
    def verifica_senha(self, senha):
        return self._senha == senha
